applications/MultilevelMonteCarloLaplacianApplication/custom_elements/my_laplacian_element.h

```diff
@@ -0,0 +1,412 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    , KratosAppGenerator
+//
+
+#if !defined(KRATOS_MY_LAPLACIAN_ELEMENT_H_INCLUDED )
+#define KRATOS_MY_LAPLACIAN_ELEMENT_H_INCLUDED
+
+
+// System includes
+#include <string>
+#include <iostream>
+#include <algorithm>
+
+// External includes
+
+
+// Project includes
+#include "includes/element.h"
+#include "my_multilevel_monte_carlo_laplacian_application_variables.h"
+
+
+namespace Kratos
+{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+
+class MyLaplacianElement : public Element {
```

Comment:
do you need this?
Or is is this the dummy?

Reply:
Good morning @philbucher 
The application is the implementation of the Multilevel Monte Carlo algorithm, and in this specific application a Laplacian is solved, so the element I created discretizes the laplacian.
As a matter of fact I can import the laplacian element from the convection diffusion application.